### ISSUE TITLE
feat: Capacitor 6 Support

### ISSUE THREAD

**Issue by rricamar** [Cap-go/capacitor-updater#357]
### Description

- `Capacitor 6` has been announced:
https://ionic.io/blog/announcing-capacitor-6-0

- Updating Plugins to `Capacitor 6`:
https://capacitorjs.com/docs/updating/plugins/6-0

- General guide for any project using `Capacitor 6` (not a plugin itself):
https://capacitorjs.com/docs/updating/6-0

🛑 Right now is not possible to perform an `npm i` command due to dependency conflict:

```bash
npm ERR! While resolving: [project-name]
npm ERR! Found: @capacitor/core@6.0.0
npm ERR! node_modules/@capacitor/core
npm ERR!   @capacitor/core@"^6.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @capacitor/core@"^5.0.0" from @capgo/capacitor-updater@5.9.4
npm ERR! node_modules/@capgo/capacitor-updater
npm ERR!   @capgo/capacitor-updater@"^5.9.4" from the root project
```

Thanks in advance 🙏

**Comment by HatemsherifMF** [comment]
Finally a new release that supports capacitor 6 has been released.

**Comment by rricamar** [comment]
> Finally a new release that supports capacitor 6 has been released.

It seems so, since latest published version (`v6.0.2`) includes: [feat!: upgrade to capacitor v6 · Cap-go/capacitor-updater@e3f761e](https://github.com/Cap-go/capacitor-updater/commit/e3f761e1497f65bce44b88ceb0a3269b5951a5ab)

I will give it a try 👍

**Comment by rricamar** [comment]
> > Finally a new release that supports capacitor 6 has been released.
> 
> It seems so, since latest published version (`v6.0.2`) includes: [feat!: upgrade to capacitor v6 · Cap-go/capacitor-updater@e3f761e](https://github.com/Cap-go/capacitor-updater/commit/e3f761e1497f65bce44b88ceb0a3269b5951a5ab)
> 
> I will give it a try 👍

Tested `v6.0.2` quickly with no `npm` errors. The installation was clean and the plugin is working as expected. Marking the issue as resolved ❤️

**Comment by HatemsherifMF** [comment]
@rricamar  is it working as expected with you ? 

it's my first time to try the plugin & I'm having issues only on IOS, after successfully updating the bundle everything seems to be fine. but after i close the app from memory and open it again, the initial bundle is set instead of the last successful one.

**Comment by rricamar** [comment]
> @rricamar is it working as expected with you ?
> 
> it's my first time to try the plugin & I'm having issues only on IOS, after successfully updating the bundle everything seems to be fine. but after i close the app from memory and open it again, the initial bundle is set instead of the last successful one.
> 
> While on Android it's working as expected.

I was going to reply that I had the same issue, but I've already seen that you've mentioned it's fixed in `v6.0.4` 🚀   (https://github.com/Cap-go/capacitor-updater/issues/362)
